### PR TITLE
Fix production limiting in specific dissipation rate equation for SSTDES

### PR DESCRIPTION
**Pull request by rcknaus**
* The kinetic energy production term in the specific dissipation rate was always getting clipped by ten times the RANS dissipation rate instead of ten times the DES dissipation rate
* Small change.  It'll only effect clipping in one term of the equation.
* I reblessed the tests and added back the "_Input" elem.  I think the NREL machines won't show diffs, but they might.
* See https://github.com/NaluCFD/Nalu/pull/462 for the corresponding NaluCFD change

### CODE REVIEW

File: reg_tests/CTestList.cmake [hunk 1]
```diff
@@ -23,6 +23,13 @@ function(add_test_r_rst testname np)
     file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/test_files/${testname})
 endfunction(add_test_r_rst)
 
+# Regression test with input
+function(add_test_r_inp testname np)
```
Comment (sayerhs) [inline reg_tests/CTestList.cmake]: I like the solution in NaluCFD/Nalu#462. Is there a reason to not rename `*_Input.i` to `*_rst.i`?

File: src/SpecificDissipationRateEquationSystem.C [hunk 2]
```diff
@@ -349,10 +358,18 @@ SpecificDissipationRateEquationSystem::register_interior_algorithm(
         (partTopo, *this, activeKernels, "sst",
          realm_.bulk_data(), *realm_.solutionOptions_, dataPreReqs, false);
 
+      build_topo_kernel_if_requested<SpecificDissipationRateSSTDESSrcElemKernel>
+        (partTopo, *this, activeKernels, "sst",
```
Comment (sayerhs) [inline src/SpecificDissipationRateEquationSystem.C]: This should be `"sst_des"`?

**Comment by sayerhs** [comment]
Thanks @rcknaus, good catch!

File: reg_tests/CTestList.cmake (hunk 1 above)
Comment (rcknaus) [reply reg_tests/CTestList.cmake]: Well, it's not a restart so I thought it was better to keep it named something separate.  Switching it to `_rst` will work too and it won't require the extra block in the CTestList file, but it's less clear what the test is doing.

**Comment by jamelvin** [comment]
Looks good!

File: src/node_kernels/SDRSSTDESNodeKernel.C [hunk 3]
```diff
@@ -0,0 +1,120 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+#include "node_kernels/SDRSSTDESNodeKernel.h"
```
Comment (jrood-nrel) [inline src/node_kernels/SDRSSTDESNodeKernel.C]: @rcknaus I think this file is missing in this pull request. When building, I'm getting an error.

Comment (rcknaus) [reply src/node_kernels/SDRSSTDESNodeKernel.C]: I forgot to include it.  I just pushed a fix to master.

**Pull request merged**